Smartling.API/Model/SampleAuditLogBuilder.cs

```diff
@@ -1,63 +1,66 @@
 ﻿namespace Smartling.Api.Model
 {
-  public class AuditLogBuilder
+  public class SampleAuditLogBuilder
```

Comment:
I like idea with the sample. And i suspect you will use this class in sitecore. I asked in [542](https://github.com/Smartling/sitecore-localization-module/issues/542) to create a method that accepts any key:value pair. I think it's a good time to do it now since you will publish new SDK version.
But feel free do it another PR.